vignettes/articles/04-advanced-data-settings.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "04 - Improve stress test by providing further project specific data"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignettes describes in detail how an analysis can be configured further by
+providing further custom data. The configurations are not necessary but, in case
+required information is available, may help you improve the accuracy of your
+results.
+
```

Comment:
Suggestion: This vignette describes in detail how an analysis can be configured further by
providing more custom data. The configurations are not required but, in case
additional information is available, may help you improve the accuracy of your
results.

Reply:
done